where-to-travel/src/main/webapp/script.js

```diff
@@ -80,6 +81,18 @@ script.async = true;
 
 document.head.appendChild(script);
 
+$('.multi-select-pill').click(function () {
+  let pillText = $(this).text();
+  if(pillText === selectedPillText) {
```

Comment:
Where is selectedPillText defined? Is this by any chance supposed to be replaced with placeType?

Reply:
Yes it is supposed to be placeType, whoops

Reply:
Resolved with c493fdc